mmedit/datasets/pipelines/blur_kernels.py

```diff
@@ -0,0 +1,519 @@
+# This code is referenced from BasicSR with modifications.
+# Reference: https://github.com/xinntao/BasicSR/blob/master/basicsr/data/degradations.py  # noqa
+import random
+
+import numpy as np
+from scipy import special
+
+np.seterr(divide='ignore', invalid='ignore')
```

Comment:
When using the sinc kernel, line 405 results in NaN owing to the division by zero. But that particular point is overwritten in line 411. This line is just used to avoid this warning.

Reply:
Okay, it should be localized to that function. Please use np.geterr to store original settings and restore them at the end of the function.